src/server/JasmineGraphInstanceService.cpp

```diff
@@ -80,4301 +144,5243 @@ void *instanceservicesession(void *dummyPt) {
-            // pthread_mutex_unlock(&file_lock);
-            string result = "1";
-            write(connFd, result.c_str(), result.size());
-            instance_logger.log("Sent : " + result, "info");
-        } else if (line.compare(JasmineGraphInstanceProtocol::DELETE_GRAPH_FRAGMENT) == 0) {
-            // Conditional block for deleting all graph fragments when protocol is used
-            instance_logger.log("Received : " + JasmineGraphInstanceProtocol::DELETE_GRAPH_FRAGMENT, "info");
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            // Read the message
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            // Get graph ID from message
-            string graphID = (data);
-            graphID = Utils::trim_copy(graphID, " \f\n\r\t\v");
-            instance_logger.log("Received Graph ID: " + graphID, "info");
-            // Method call for graph fragment deletion
-            removeGraphFragments(graphID);
-            // pthread_mutex_lock(&file_lock);
-            // TODO :: Update catalog file
-            // pthread_mutex_unlock(&file_lock);
-            string result = "1";
-            write(connFd, result.c_str(), result.size());
-            instance_logger.log("Sent : " + result, "info");
-        } else if (line.compare(JasmineGraphInstanceProtocol::DP_CENTRALSTORE) == 0) {
-            instance_logger.log("Received : DP_CENTRALSTORE from server", "info");
-            write(connFd, JasmineGraphInstanceProtocol::OK.c_str(), JasmineGraphInstanceProtocol::OK.size());
-            instance_logger.log("Sent : " + JasmineGraphInstanceProtocol::OK, "info");
-            bzero(data, INSTANCE_DATA_LENGTH + 1);
-            read(connFd, data, INSTANCE_DATA_LENGTH);
-            string graphID = (data);
-            graphID = Utils::trim_copy(graphID, " \f\n\r\t\v");
-            instance_logger.log("Received Graph ID: " + graphID, "info");
+JasmineGraphIncrementalLocalStore *JasmineGraphInstanceService::loadStreamingStore(
+    std::string graphId, std::string partitionId,
+    std::map<std::string, JasmineGraphIncrementalLocalStore *> &graphDBMapStreamingStores) {
+    std::string graphIdentifier = graphId + "_" + partitionId;
+    instance_logger.log("###INSTANCE### Loading streaming Store for" + graphIdentifier + " : Started", "info");
+    std::string folderLocation = Utils::getJasmineGraphProperty("org.jasminegraph.server.instance.datafolder");
+    JasmineGraphIncrementalLocalStore *jasmineGraphStreamingLocalStore =
+        new JasmineGraphIncrementalLocalStore(atoi(graphId.c_str()), atoi(partitionId.c_str()));
+    graphDBMapStreamingStores.insert(std::make_pair(graphIdentifier, jasmineGraphStreamingLocalStore));
+    auto sg = graphDBMapStreamingStores.find(graphIdentifier);
```

Comment:
Although the variable `sg` assigned it is never used. There is no point of assignment.

Reply:
Removed it in next PR

Reply:
Ok